Controllers/ColourController.cs

```diff
@@ -16,7 +16,6 @@ public ColourController(IColourService colourService)
             _colourService = colourService;
         }
 
-        // GET
         [HttpGet]
         public IEnumerable<Colour> Get()
```

Comment:
Seeing as this is a controller, it might be better for this method to return an `IActionResult`.
You can then do something like:
```
var colours = _colourService.Get();
return this.Ok(colours);
```

Reply:
Another pro of using `IActionResult` is having access to `this.NotFound()`, `this.BadRequest()` etc.
More info [here](https://docs.microsoft.com/en-us/aspnet/core/web-api/action-return-types?view=aspnetcore-2.2)